server_tests/test/clientTests.js

```diff
@@ -791,514 +736,468 @@ describe("RUST API TEST SUITE", function() {
                                 'sleepDelay', 'offDelay', 'hdr', 'exposureBracket', 'gyro', 'gps',
                                 'imageStabilization', '_bublVideoFileFormat'];
 
-        before( function(done) {
-            testClient.startSession()
-            .then( Comparison.catchExceptions(done, function(res) {
+        before( function() {
+            return testClient.startSession()
+            .then( function onSuccess (res) {
                 sessionId = res.body.results.sessionId;
                 Comparison.oscSessionOpOutput(res, {'sessionId': sessionId});
-                done();
-            }));
+            }).catch(shouldNotGetHere);
```

Comment:
`catch()` is inconsistent with the usual approach of using the second error handler parameter.